mlflow/pyfunc/model.py

```diff
@@ -281,6 +283,8 @@ def _save_model_with_class_artifacts_params(
         mlflow_model: The model to which to add the ``mlflow.pyfunc`` flavor.
         model_config: The model configuration for the flavor. Model configuration is available
             during model loading time.
```

Comment:
Seems like we're accidentally removing the experimental note from `model_config`. Can we add it back?

```
            .. Note:: Experimental: This parameter may change or be removed in a future release
                without warning.
```